build.psm1

```diff
@@ -2377,6 +2377,7 @@ function Start-CrossGen {
             "Microsoft.PowerShell.CoreCLR.Eventing.dll",
```

Comment:
@iSazonov, your last commit had 4 failures in `PowerShell-CI-windows`
Validate simple Add-LocalGroupMember.Can add local user to group by name
```powershell
You cannot call a method on a null-valued expression.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.LocalAccounts\Pester.Command.Cmdlets.LocalAccounts.LocalGroupMember.Tests.ps1: line 90
90:             $result.Name.EndsWith("TestUser1") | Should -BeTrue

```
Validate simple Get-LocalGroupMember.Can get a local group member by name
```powershell
You cannot call a method on a null-valued expression.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.LocalAccounts\Pester.Command.Cmdlets.LocalAccounts.LocalGroupMember.Tests.ps1: line 265
265:             $result.Name.EndsWith("TestUserGet1") | Should -BeTrue

```
Validate simple Rename-LocalUser.Can Rename-LocalUser by only name
```powershell
Expected strings to be the same, but they were different.
String lengths are both 15.
Strings differ at index 14.
Expected: 'TestUserRename2'
But was:  'TestUserRename1'
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.LocalAccounts\Pester.Command.Cmdlets.LocalAccounts.LocalUser.Tests.ps1: line 845
845:             $result.Name | Should -BeExactly TestUserRename2
```
Validate simple Remove-LocalUser.Can Remove-LocalUser with only name
```powershell
Expected strings to be the same, but they were different.
Expected length: 62
Actual length:   63
Strings differ at index 0.
Expected: 'UserNotFou...'
But was:  'GroupNotFo...'
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.LocalAccounts\Pester.Command.Cmdlets.LocalAccounts.LocalUser.Tests.ps1: line 33
33:         $_.FullyQualifiedErrorId | Should -Be $expectedFqeid
```